src/androidUtils.ts

```diff
@@ -1,40 +1,35 @@
-/**
- * @format
- * @flow strict-local
- */
 import {ANDROID_DISPLAY, ANDROID_MODE} from './constants';
-import pickers from './picker';
-import type {AndroidNativeProps, DateTimePickerResult} from './types';
+import pickers from './picker.android';
+import type {
+  AndroidNativeProps,
+  ButtonType,
+  DateTimePickerResult,
+} from './types';
 import {sharedPropsValidation} from './utils';
 import invariant from 'invariant';
-import {processColor} from 'react-native';
 
 type Timestamp = number;
 
-type ProcessedButton = {
```

Comment:
@vonovak ProcessedButton and ButtonType had mismatching types (title vs label), I presume ButtonType is the type we are wanting to move forward with to match the expected props for `positive`, `negative`, and `neutralButton`

Reply:
we want to move forward in a way that maintains the public api (we don't want a breaking change), so keep `label` exposed. Then we need to verify that it keeps working, i.e. that passing the prop actually sets the button title.

Reply:
So the public facing type should be `ButtonType` and the privately used one is the `ProcessedButton` then correct? As title should be passed to the NativeModule for the Android settings but the users JSX should define the buttons as having a label prop, correct?

Reply:
yes, correct! :)

Reply:
Should the `AndroidNativeProps` in the types file have the `ProcessedButton` or `ButtonType` type on it then? Currently it has the `ButtonType` type set to it, but based on these comments it sounds like we expect to pass the processed color as the `textColor` value to the native side of things.

Reply:
let me put it this way - the public api stays the same as ti is now: user provides `label` and `textColor`. We take that, process the color, and pass the processed data over to native. Native takes that and sets up the corresponding buttons. If it works at runtime on a emulator AND typescript is happy, then we're done! That's the way it should work 🙂 . Hope it makes sense :)

Reply:
I think I can't tell how exactly they should line up until I have a proper look at them so let me look at it. Tomorrow probably :)

Reply:
Thank you for jumping in to help! Please feel free to PM me on Twitter/X or somewhere if you would like as well to talk about some of the changes or things I may have missed :)

Reply:
@Coffeegerm fixed. 

Btw, for errors such as

```
example/SegmentedControl.tsx:4:32 - error TS7017: Element implicitly has an 'any' type because type 'typeof globalThis' has no index signature.

const isFabricEnabled = global.nativeFabricUIManager !== null;
```

don't try to type those things - just put a `@ts-expect-error` there. Thank you

Reply:
Thanks so much for hopping on that fix! This was what my assumption was with changing the prop from `title` to `label` but was confusing myself along the way.

Reply:
Yeah, flow doesn't work as well as TS with platform suffixes in file names which I think is why flow did not report an error.